services/tenant-ui/frontend/src/components/common/ValidatedField.vue

```diff
@@ -0,0 +1,84 @@
+<template>
```

Comment:
Yeah this is perfect, we've needed this for a **long** time as the validation syntax is so verbose all over forms all over the app. 
We should expand to use this everywhere we can (and might need additional ones for other form controls) as we go through other components

The one thing in my mind on this also, is if we expand this common component to be used, do you think it's written in such a way that we could swap out the validation library if ever needed?
**Not saying we need to do this**, but Primevue when we first built this app had Vuelidate in it's documentation as the "suggested" vue validation framework, and some point recently they've changed it to VeeValidate

https://primevue.org/inputswitch/#veevalidate

Reply:
This component would need to be changed obliviously but the best part is it only has to be change in one place. Then the validation rules for each form would need to be changed as well which would be quite a bit of work. The nice thing is though the all the form components using it would become obvious because of typescript. Once you changed the type to veevalidate.

There is other places this type of thing can be done but I figure we can just do them when we see them and when we have time.

Reply:
Yeah I think it's knock them down as we change other related things.

Ok, validation library thing noted, as I said not thinking we should change but yeah it's good it could be more contained if we did. Mostly annoyed that the Primevue docs changed supported at some point.